src/metabase/driver.clj

```diff
@@ -299,6 +299,11 @@
   (when engine
     (contains? (set (keys (available-drivers))) (keyword engine))))
 
+(defn driver-supports?
+  "Predicate function that tests if a driver supports a given feature."
```

Comment:
The question mark at the end of the function name implies that it's a predicate function, I don't think it's really necessary to include that in the docstr